### ISSUE TITLE
[Suggestion] Eldritch Spire Exterior

### ISSUE THREAD

**Issue by rougedarkdragon** [TheCodex6824/ThaumicAugmentation#345]
Hi, the eldritch spire is one of my favorite parts of this mod, and it really makes thaumcraft complete, but the exterior is a bit (no offense) unrefined. Maybe you could give it some dramatic exterior (the spaces in between the walls and room are just filled with blocks). If you want I could try to do it but I might not know what kind of lore surrounds it.

**Comment by TheCodex6824** [comment]
Yeah, I'm not happy with how it is right now either. At some point (tm) I'll get to improving it - I have some ideas for it. The main problem is just not having much time anymore for it :(

**Comment by rougedarkdragon** [comment]
Okay. I can't wait to see it whenever you make it!
